services/index.js

```diff
@@ -6,3 +6,5 @@ export * from "./getNavigation";
 export * from "./getPropsTable";
 export * from "./getDependencies";
 export * from "./getPackageData";
+
+export const exemptList = ["image", "text"];
```

Comment:
Lists the components that we haven't built yet but that have some doc pages. This was the easiest way to check them, without reading files.